apps/server/src/modules/user-import/controller/api-test/import-user-populate.api.spec.ts

```diff
@@ -113,7 +113,8 @@ describe('ImportUser Controller Populate (API)', () => {
 			const setup = async () => {
 				const { account, school, system } = await authenticatedUser([Permission.IMPORT_USER_MIGRATE], [], false);
 				const loggedInClient = await testApiClient.login(account);
-				userImportFeatures.userMigrationSystemId = system.id;
+				const config: ServerConfig = serverConfig();
+				config.FEATURE_USER_MIGRATION_SYSTEM_ID = system.id;
```

Comment:
As you create the config in each test again, you could delete the setConfig in line 43 and the before each completely if you add here: config.FEATURE_USER_MIGRATION_ENABLED = true;
Though i like the structure to have a standard config setup in each test....